{{cookiecutter.github_repository}}/provisioner/site.yml

```diff
@@ -28,5 +29,6 @@
     - nginx
     - supervisor
     - project_data
+    - { role: nodejs, when: sass_with_django_compressor }
```

Comment:
Since we have `node` role as dependency in `project_data` we won't need it here.